clients/privacy-center/package-lock.json

```diff
@@ -8909,10 +8872,9 @@
       }
     },
     "node_modules/minimist": {
-      "version": "1.2.5",
-      "resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.5.tgz",
-      "integrity": "sha512-FM9nNUYrRBAELZQT3xeZQ7fmMOBg6nWNmJKTcgsJeaLstP/UODVpGsr5OhXhhXg6f+qtJ8uiZ+PUxkDWcgIXLw==",
-      "license": "MIT"
```

Comment:
I don't see why the license should be removed. Minimist 1.2.6 [still uses the MIT license](https://github.com/substack/minimist/blob/master/LICENSE).

Reply:
I have gone ahead and added it back. I have check that running `npm i` does not override addition.